### PR TITLE
Fix vars for cidev

### PR DESCRIPTION
**Pull request by markpit**
Adding restricted_word_web_port and chs_url into cidev config

### CODE REVIEW

File: terraform/groups/ecs-service/locals.tf [hunk 1]
```diff
@@ -46,9 +46,11 @@ locals {
   ]
 
   task_environment = [
-    { "name": "LOG_LEVEL", "value": "${var.log_level}" },
+    { "name": "CHS_URL", "value": "${var.chs_url}" },
```
Comment (markpit) [inline terraform/groups/ecs-service/locals.tf:49]: changed to alphabetical order as well

File: terraform/groups/ecs-service/variables.tf [hunk 2]
```diff
@@ -67,7 +61,22 @@ variable "cookie_name" {
...
+variable "log_level" {
+  default     = "info"
+  type        = string
+  description = "The log level for services to use: trace, debug, info or error"
+}
 variable "secure_cookie" {
   type        = string
   default     = "0"
 }
+
```
Comment (mwestacott) [inline terraform/groups/ecs-service/variables.tf:73]: Minor, but no empty line needed here.

File: terraform/groups/ecs-service/variables.tf [hunk 3]
```diff
@@ -67,7 +61,22 @@ variable "cookie_name" {
...
+variable "restricted_word_web_port" {
+  type        = string
+  default     = "3000"
+}
+variable "restricted_word_web_version" {
```
Comment (mwestacott) [inline terraform/groups/ecs-service/variables.tf:78]: Was this missing too?

**Review by tobafolaranmi-ch — APPROVED** [review]
LGTM

File: terraform/groups/ecs-service/variables.tf (hunk 2 above)
Comment (markpit) [reply terraform/groups/ecs-service/variables.tf]: gone

**Review by dlloyd1-ch — APPROVED** [review]

**Pull request merged**